spec/domain/messages/bulk_mail_dispatch_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, CVP Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+require 'spec_helper'
+
+describe Messages::BulkMailDispatch do
+  let(:message) { messages(:mail) }
+  let(:top_leader) { people(:top_leader) }
+  let(:dispatch) { described_class.new(message) }
+
+  let(:address_list) {
```

Comment:
only add variables here that are used in multiple spec blocks. 
if only used in one, just create a variable inside the block it-self